### PR TITLE
Workload test procedures

### PR DESCRIPTION
**Pull request by DeV1doR**

### CODE REVIEW

File: scripts/Makefile [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+THREADS=1
+TIME=30
+
+ts := $(shell /bin/date "+%Y%m%d%H%M%S")
+
+help:
+	@echo "$$HELPBODY"
+
+load_transfer:
+	wrk -c ${CLIENTS} -t ${THREADS} -d ${TIME} --latency -s ./token_post.lua ${REST_API_URL}/api/v1/token
```
Comment (eugene-babichenko) [inline scripts/Makefile]: A little note on this test: as Sawtooth framework can execute independent transactions in parallel this test runs on the worst case (all transactions are dependent).

File: scripts/Makefile [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+load_get_balance:
+	wrk -c ${CLIENTS} -t ${THREADS} -d ${TIME} --latency ${REST_API_URL}/api/v1/token/03fcc39066206a010d4a9b04ad3ebe6fbdcdd0ac8016973b005ede707322a9aa18
+
+load_batch_status:
+	wrk -c ${CLIENTS} -t ${THREADS} -d ${TIME} --latency ${REST_API_URL}/api/v1/batch_status/8620c54ed41eae6a61eef1374536abf0ea17ff7923d74bdae0b1f9a5b621b4042c085e62ea01260435a9769ab02d785f5088bf2873497ca38b128cb086cdcc1d
+
+load_user_pub_keys:
+	wrk -c ${CLIENTS} -t ${THREADS} -d ${TIME} --latency ${REST_API_URL}/api/v1/user/03fcc39066206a010d4a9b04ad3ebe6fbdcdd0ac8016973b005ede707322a9aa18/pub_keys
+
+load_certificate_creation:
```
Comment (eugene-babichenko) [inline scripts/Makefile]: Same here: the worst case. We need an opposite test which runs on a set of different users.

Comment (eugene-babichenko) [reply scripts/Makefile]: same for all test cases down there.

File: scripts/Makefile (hunk 1 above)
Comment (DeV1doR) [reply scripts/Makefile]: @eugene-babichenko are we testing sawtooth or our node? In current implementation we have only one transaction per batch, maybe need to reimplement the functionally and after get back for testing this? Or I miss understood the problem.

**Pull request merged**